presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -113,11 +112,13 @@ public void shouldExecuteQueryWithSelectedCatalogAndSchema()
     public void shouldSetTimezone()
             throws SQLException
     {
-        String timeZoneId = "Indian/Kerguelen";
-        connection.setTimeZoneId(timeZoneId);
-        try (Statement statement = connection.createStatement()) {
-            QueryResult result = queryResult(statement, "select current_timezone()");
-            assertThat(result).contains(row(timeZoneId));
+        if (connection instanceof PrestoConnection) {
```

Comment:
http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/Preconditions.html#checkState(boolean)
That would throw an exception when it is run with Simba's driver, which would cause the test to fail .  Is that what you want?